### PR TITLE
["""MODULAR"""] Re-criminalizes the augmented eyes, by moving them behind alien medical research.

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request
what it says on the tin.
## How This Contributes To The Skyrat Roleplay Experience

Access to information is the most powerful tool in any game, especially so for SS13, and either eyes makes stealth or else wise subtlety pointless.

## Changelog

:cl:

balance: Moved Xray+Thermal eyes behind the alien biological research node.
/:cl:

closes #11202

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -1256,17 +1256,15 @@
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 
-/datum/techweb_node/combat_cyber_implants
+/datum/techweb_node/combat_cyber_implants // SKYRAT EDIT - REMOVED: X-rays+Thermals from here
 	id = "combat_cyber_implants"
 	display_name = "Combat Cybernetic Implants"
 	description = "Military grade combat implants to improve performance."
 	prereq_ids = list("adv_cyber_implants","weaponry","NVGtech","high_efficiency")
 	design_ids = list(
```
Comment (GoldenAlpharex) [inline code/modules/research/techweb/all_nodes.dm:1265]: Move the Skyrat edit to like, the line before this one or something like that.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by Higgin** [comment]
It'd be one thing if they were getting given out every round hard and early - but they aren't, and I've barely seen folks getting them when they can. 

Being able to see a place does not necessarily give you access to that place, and these seem to be pushed back late enough in the round by their current tech reqs to give people a good window of time to do stealth shit before they come into play. They also don't strictly prevent stealth or social subtlety at all.

Getting more information of any sort requires time and effort - be that adding/upgrading cameras, checking every locker, etc. This just flattens the top-end of information one person can get (after having had to work with somebody else to get them installed, mind you.) 

It feels both unnecessary and like it takes away a sound, cool, late-round tool - which people have to work for and which, in the case of people hiding in dorm rooms/super-nooks after shooting Ian, allows the crew to put pressure back on them and restart the conflict (assuming they even know who did it.)

**Comment by SpookyTheFox** [comment]
As I said on discord, this should not be adjusted. It's in exactly the right spot in the research tree as-is.

**Comment by RatFromTheJungle** [comment]
> Being able to see a place does not necessarily give you access to that place, and these seem to be pushed back late enough in the round by their current tech reqs to give people a good window of time to do stealth shit before they come into play. They also don't strictly prevent stealth or social subtlety at all.

It's actually remarkably early in research, and as long as you spend a few moments looking at the tree and it's pre-reqs, very afford sub even 50m, while still getting t4's

It does prevent stealth, because you can see people through the walls, and x-ray lets you see items/machines in the room as-well, meaning any in-place build, even not antag is instantly made irrelevant is it's so much as on-screen to a seccie, meaning antags are sandwiched into the loud-only objective-to-objective rush because sitting still will get them caught, especially if the AI or a seccie has seen them through a wall at any point.
> Getting more information of any sort requires time and effort - be that adding/upgrading cameras, checking every locker, etc. This just flattens the top-end of information one person can get (after having had to work with somebody else to get them installed, mind you.)

Time and effort which walking around after getting an implant doesn't require, and most other means can just be disabled, nothing stops you from just taking apart the cams around where you're gonna be, but short of spamming EMPs over and over, there's nothing you can do to stop from getting seen through the wall by a seccie short of a sixth-sense

**Comment by GoldenAlpharex** [comment]
> As I said on discord, this should not be adjusted. It's in exactly the right spot in the research tree as-is.

I disagree. TG's balance isn't the same as ours, and as Jungle said in the PR, this is literally the most powerful tool accessible to players without admin intervention.
I like using it, but I can still recognize that it's incredibly busted and never actually gets used by the target crowd that was given in the PR that re-enabled them, it's Security that gets these, and by that point an antag has pretty much no chance if they can't teleport, because they'll inevitably get found.
It's a nightmare to deal with for staff, too, and I'm personally of the opinion that we shouldn't have re-introduced Thermal, as there's no use to it aside from validhunting.

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply code/modules/research/techweb/all_nodes.dm:1265]: I literally meant this line here.

**Review by tf-4 — APPROVED** [review]

**Comment by SpookyTheFox** [comment]
> It's actually remarkably early in research, and as long as you spend a few moments looking at the tree and it's pre-reqs, very afford sub even 50m, while still getting t4's

Even with it reenabled, I still see many rounds that it does not get unlocked during. And of those that are, it is not used often either. And even if it does get unlocked that early, which I have never seen, it is costly enough to the station that they are putting themselves at a disadvantage in other areas.

Do you have any alternative proposals to reward the station with diligent research, that fulfill the same end goals?

> I disagree. TG's balance isn't the same as ours, and as Jungle said in the PR, this is literally the most powerful tool accessible to players without admin intervention.

On the contrary. We have traitors that are largely following TG progression. Do you have any alternative solutions that address the points I made when I re-enabled it?

I went through the process, and discussed the matter on discord before I made the PR. There were no objections at that time. I had unanimous agreement before I made the PR in the first place.

I do not think we will reach an agreement without major compromise, and that is not what this PR is offering.

**Comment by SpookyTheFox** [comment]
> It's a nightmare to deal with for staff, too, and I'm personally of the opinion that we shouldn't have re-introduced Thermal, as there's no use to it aside from validhunting.

Interesting opinion. Perhaps we need to have a wider discussion on crew vs antag involvement and expectations. And I would like to hear more about this staff issue, I haven't heard anything about it in the 12 days it's been enabled.

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (RatFromTheJungle) [reply code/modules/research/techweb/all_nodes.dm:1265]: 👍

**Comment by softcerv** [comment]
I've brought this up on the discord before, but I worry that having thermals be widely accessible will make it easier for bad actors to target and break up people ERPing. While it's not removing them entirety, I think this is a good step in making scenarios like this far less likely to happen.

**Comment by Higgin** [comment]
> It does prevent stealth, because you can see people through the walls, and x-ray lets you see items/machines in the room as-well, meaning any in-place build, even not antag is instantly made irrelevant is it's so much as on-screen to a seccie, meaning antags are sandwiched into the loud-only objective-to-objective rush because sitting still will get them caught, especially if the AI or a seccie has seen them through a wall at any point.

disguise, unscrewing your maint comms console, chilling in a locker or box, accepting that you might be getting hunted after going loud or getting caught out, using social stealth, using a stealth implant, etc... all options for dealing with this, it's really not as big of an impediment as you're framing it to be.

The thing about time is 'sub-50 minutes with T4s' assumes a beeline for thermals/x-rays, too, that does not happen in practice - it didn't even happen after MCRs came out and everybody was horny for those (in no small part because of being gated by experiments, but also cost.) If they're ready at 50 minutes, you still have to add in the time and coordination it takes for somebody to go to robotics/med to get them - you have to assume that both parties aren't already busy or can drop everything they're doing to do discretionary implants that hardly happen as-is due to alert policy.

> Time and effort which walking around after getting an implant doesn't require, and most other means can just be disabled, nothing stops you from just taking apart the cams around where you're gonna be, but short of spamming EMPs over and over, there's nothing you can do to stop from getting seen through the wall by a seccie short of a sixth-sense

if the issue is counterplay, would it help to reduce the range on x-ray and make both types of eyes much weaker to EMP

it doesn't require as much effort (except to be walking around and looking) after installation for these things to be beneficial, no, and that's why they're an upgrade - even still, they're an upgrade most people aren't going for or approaching at this point in the game until much later in the round + when there are known issues.

**Comment by tf-4** [comment]
> I do not think we will reach an agreement without major compromise, and that is not what this PR is offering.

That's exactly what this PR is, a compromise between having it and not having it.

**Comment by SpookyTheFox** [comment]
> > I do not think we will reach an agreement without major compromise, and that is not what this PR is offering.
> 
> That's exactly what this PR is, a compromise between having it and not having it.

However, it does not address the reasons I brought up to re-enable it. We need a predictable, and hopefully time based, not RNG, counter to the situations I brought up that necessitated re-enabling it.

**Comment by RatFromTheJungle** [comment]
> > > I do not think we will reach an agreement without major compromise, and that is not what this PR is offering.
> > 
> > 
> > That's exactly what this PR is, a compromise between having it and not having it.
> 
> However, it does not address the reasons I brought up to re-enable it. We need a predictable, and hopefully time based, not RNG, counter to the situations I brought up that necessitated re-enabling it.

"necessitated" is a bit steep of a statement, it's closer to 'I really wanted it," most of the issues you brought up were very mild, including finding bodies, the stations are notably deep and by just hugging the right wall when walking near where they're lost, you'll usually find them.

If you want to make a PR adjusting other items to be more inline, that's fine, but none of the issues brought up in the original PR's desc justify the presence of one of, if not the strongest abilities in game, espiecally mass-printably, and reliably attainable.

**Review by KathrinBailey — APPROVED** [review]
I'd straight-up merge this if it weren't for people would start complaining it's a sub-24hr merge.
This compromise is more than fair.

If you want the Valid Detectors™ to be easily accessible again, there's nothing to stop anyone adding a downside to using them and moving which techweb they're under again.

**Comment by MassacreDemigirl** [comment]
>by that point an antag has pretty much no chance if they can't teleport, because they'll inevitably get found.

Also if they can teleport and not a full screen range, they will get busted as well

**Comment by hugmeorelse** [comment]
This isn't greenlit.

**Comment by Arturlang** [comment]
Moving it to alien research will essentially makes this inaccesible for most if not all situations, unless you get very, very lucky or abuse a exploit. Essentially deleting it without actually doing so.

**Review by KathrinBailey — APPROVED** [review]
Put to an internal maintainer vote between this and #11202.
4 out of 5 maintainers votes for this, whilst 1 out of 5 maintainers hasn't yet voted.

**Comment by projectkepler-ru** [comment]
oh if only you know

**Comment by MassacreDemigirl** [comment]
commenting at a 2y old pull

**Pull request merged**